charts/sentry/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 name: sentry
 description: A Helm chart for Kubernetes
 type: application
-version: 23.11.0
+version: 23.11.1
```

Comment:
You shouldn't update that.

Reply:
@Mokto I thought so but the lint test seems to indicate otherwise 
![Screenshot 2024-07-04 at 6 05 53 PM](https://github.com/sentry-kubernetes/charts/assets/11012686/3b3535ce-7521-46b8-a710-c32a6c0b3ea7)
 🤔  Am I understanding it wrong?